third_party/terraform/tests/resource_compute_forwarding_rule_test.go.erb

```diff
@@ -182,3 +182,59 @@ resource "google_compute_forwarding_rule" "foobar" {
 }
 `, poolName, ruleName)
 }
+<% unless version == 'ga' -%>
+func TestAccComputeForwardingRule_mirror(t *testing.T) {
+	t.Parallel()
+
+	backendName := fmt.Sprintf("tf-%s", acctest.RandString(10))
+	healthcheckName := fmt.Sprintf("tf-%s", acctest.RandString(10))
+	ruleName := fmt.Sprintf("tf-%s", acctest.RandString(10))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeForwardingRuleDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeForwardingRule_mirror(backendName, healthcheck
+				Name, ruleName),
```

Comment:
Looks like there is an extra newline here that is causing issues

Reply:
Not sure where that came from. I've removed it